### PR TITLE
Removing hard-coded afflictions for phlegmatic inundate

### PR DESCRIPTION
**Pull request by jgh713**
Removing unknownany and hard-coded slickness from the inundate function, per #200

### CODE REVIEW

File: raw-svo.dict.lua [hunk 1]
```diff
@@ -1000,24 +1000,11 @@ dict = {
 
         anorexia 50% time
         slickness 8+
+		
+		Previous information is old. Now gives between one and four afflictions from the following: lethargy, slickness, anorexia, weariness.
+		Afflictions not hidden by gmcp, so removed from the inundated function.
```
Comment (keneanung) [inline raw-svo.dict.lua]: Why don't you remove the old information then?

**Comment by jgh713** [comment]
I was told it was intended.

File: raw-svo.dict.lua (hunk 1 above)
Comment (jgh713) [inline raw-svo.dict.lua]: Dunno.

**Comment by keneanung** [comment]
Told by a player or was that a bug report response?

**Comment by jgh713** [comment]
> Message #2786       Sent by Makarios
> 2017/3/11/23:29 Ah, that's intended, its not random. Inundate phlegmatic is entirely deterministic.

**Pull request merged**